SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
```

Comment:
Added this file as nice-to-have. What's really missing is a README.md file. Suggest someone else provide that.